data/tlg0013/tlg025/__cts__.xml

```diff
@@ -5,19 +5,19 @@
       
    <ti:translation projid="greekLit:perseus-eng2" urn="urn:cts:greekLit:tlg0013.tlg025.perseus-eng2" xml:lang="eng" workUrn="urn:cts:greekLit:tlg0013.tlg025">
         
-       <ti:label xml:lang="eng">Hymn 25 to the Muses and Apollo, Hesiod The Homeric hymns. And Homerica</ti:label>
+       <ti:label xml:lang="eng">Hymn 25 to the Muses and Apollo</ti:label>
         
-       <ti:description xml:lang="eng">Hymni Homerici, creator; Hesiod, creator; Homer, creator; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, editor; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator</ti:description>
+       <ti:description xml:lang="eng">Anonymous. Hesiod, The Homeric Hymns and Homerica. Evelyn-White, Hugh G., translatr. London: William Heinmann; New York: The Macmillan Co., 1914.</ti:description>
```

Comment:
typo in translator

Reply:
hi @lcerrato aw, got sloppy at the end! thanks for catching!